src/index.html

```diff
@@ -11,7 +11,208 @@
+          >
+        </div>
+        <div class="last">
+          <label for="term">
+            I accept the term of the agreement
+          </label>
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div>
+          <label for="email">
+            E-mail:
+          </label>
+          <input
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            name="email"
+            autocomplete="off"
+          >
+        </div>
+        <div class="last">
+          <label for="pass">
+            Password:
+          </label>
+          <input
+            type="password"
+            id="pass"
+            name="Password"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div>
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="cats"
+          > <label for="yes">Yes</label>
+          <input
+            type="radio"
+            id="no"
+            name="cats"
+          > <label for="no">No</label>
+        </div>
+        <div>
+          <label for="color">
+            What is your favorite color?
+          </label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+          >
+        </div>
+        <div>
+          <label for="time">
+            What time do you go to bed?
+          </label>
+          <input
+            type="time"
+            step="1"
+            id="time"
+            name="time"
+          >
+        </div>
+        <div>What are your favorite brand of cars?
+          <select name="brand" multiple>
+            <option value="Bmw">
+              BMW
+            </option>
+            <option value="Audi">
+              Audi
+            </option>
+            <option value="Lada">
+              Lada
+            </option>
+          </select>
+        </div>
+        <div class="last">
+          <label for="rate">
+            How do you rate our work?
+          </label>
+          <input
+            type="range"
+            value="0"
+            min="0"
+            max="100"
+            id="rate"
+            name="rate"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Additional info:
+        </legend>
+        <div>
+          <label for="com">
+            Comments:
+          </label>
+          <textarea
+            minlength="10"
+            maxlength="300"
+            name="Comments"
+            id="com"
+            cols="30"
+            rows="2"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            minlength="10"
            maxlength="300"
            name="Comments"
            id="com"
            cols="30"
            rows="2"
            autocomplete="off"
          ></textarea>
```
To avoid this 
![image](https://user-images.githubusercontent.com/97547824/202850356-caf9ff73-a5d6-4a6d-ae28-aa13abbff65e.png)